core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.ts

```diff
@@ -79,6 +115,38 @@ export class DotEmaComponent implements OnInit {
         });
     }
 
+    ngAfterViewInit(): void {
+        fromEvent(this.window, 'message')
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((event: MessageEvent) => {
+                this.handlePostMessage(event)();
+            });
+    }
+
+    ngOnDestroy(): void {
+        this.destroy$.next(true);
+        this.destroy$.complete();
+    }
+    /**
+     * Handle the iframe load event
+     *
+     * @param {CustomEvent} event
+     * @memberof DotEmaComponent
+     */
+    onIframeLoad(_: Event) {
+        this.loadingIframe = false;
+        // This event is destroyed when you close the dialog
```

Comment:
How is that happening? or where?

Reply:
The dialog removes the iframe from the DOM when it closes, that destroys the event listeners